packages/importer/src/config.ts

```diff
@@ -230,6 +230,11 @@ export interface OneSchemaValidationHooksToAdd {
   batch_size?: number
 }
 
+export type OneSchemaAdvancedMappingToAdd = {
```

Comment:
If/when we add other types of advanced mappings, this type will be an `|` of types.